tools/docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:18.04
+RUN apt-get update
+RUN apt-get install -y wget
+RUN apt-get install -y default-jdk
+ENV version 0.8.1
+ENV kotlin_native kotlin-native-linux-${version}
+RUN wget https://download-cf.jetbrains.com/kotlin/native/builds/releases/${version}/linux/${kotlin_native}.tar.gz
```

Comment:
Is `wget` always there?

Reply:
What do you mean with `always there`?
In Line 3 `wget` will be installed (if not already shipped by default in ubuntu what I guess).